.github/workflows/actions.yml

```diff
@@ -0,0 +1,35 @@
+name: repo-checks
+on:
+  push:
+    branches:
+      - '**' # run on push to any branch, but not pushes to tags
+  pull_request:
```

Comment:
prefer reusing our regular config ... unless this is temporary for the initial PR

Reply:
Please point us. We fake it every time. Cribbed the above from classic so that we could get branches under test.

Reply:
Yeah. Not sure where to find the regular config

Reply:
https://github.com/zendesk/stronger_parameters/blob/master/.github/workflows/actions.yml#L1-L6